Documentation/Signoffs/Communication.md

```diff
@@ -0,0 +1,92 @@
+# Communication Subsystem
+
+**Function:**
+
+![Function](../Images/Communication/Conceptual.PNG)
+
+Figure 1: Communication Subsystem
+
+The goal of this subsystem is to receive and transmit the data from the sensor subsystem to the processor.  
+
+**Constraints:**
+
+| NO. | Constraint                                                          | Origin           |
+|-----|---------------------------------------------------------------------|------------------|
+| 1   | The connection between the interceptor and sensors shall be a wireless connection           |Conceptual Design |
+| 2   | The wireless connection shall be password protected to minimize any outside party interference with the intercepting device                |Conceptual Design |
+| 3   | The connection shall operate within the federally defined frequency range of either 2.4 Ghz or 5 Ghz            |Conceptual Design |
```

Comment:
These aren't the only allowable bands. this is the wi-fi bands. And, if you choose to setup your own wifi network, that is fine. I only wanted to make sure you were aware that you aren't limited to these bands in general.